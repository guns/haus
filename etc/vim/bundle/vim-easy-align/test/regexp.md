" :source run.vim
" @a

apple;:banana::cake
data;;exchange:;format

```ruby
apple = 1 # comment not aligned
apricot = 'DAD' + 'F#AD'
banana = 'Gros Michel' # comment 2
```

