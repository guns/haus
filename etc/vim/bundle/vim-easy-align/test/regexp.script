4Gvipjyvip:EasyAlign:Pvip:EasyAlign*:Pvip:EasyAlign-:Pvip:EasyAlign * :Pvip:EasyAlign -2 :Pvip:EasyAlign -2:Pvip:EasyAlign *:Pvip:EasyAlign;Pvip:EasyAlign /;/Pvip:EasyAlign/;/Pvip:EasyAlign*/;/Pvip:EasyAlign/[:;]\+/Pvip:EasyAlign*/[:;]\+/Pvip:EasyAlign* /[:;]\+/Pvip:EasyAlign */[:;]\+/Pvip:EasyAlign   *   /[:;]\+/   Pvip:EasyAlign-/[:;]\+/Pvip:EasyAlign-2/[:;]\+/Pvip:EasyAlign/[:;]\+/{ 'l': '<<<', 'r': '>>>'}  Pvip:EasyAlign/[:;]\+/{'l':'', 's': 1}Pvip:EasyAlign *  /[:;]\+/ {'stick_to': 1, 'le': '???', 'marr': '~~~'}  Pvip:EasyAlign /../Pvip:EasyAlign*/../Pvip:EasyAlign **/[:;\+]/Pvip:EasyAlign/[:;]\+/{'rm':'____'}Pvip:EasyAlign**/[:;]\+/Pvip:EasyAlign!**/[:;]\+/Pfbjfm:EasyAlign/[:;]\+/90Gvip:EasyAlign/#/{'is':['String'],'iu':0}93Gpvip:EasyAlign*/\s*/{'l':'(', 'r': ')'}96Gpvip:EasyAlign*/[:;]\+/r3l3s1Pvip:EasyAlign*/[:;]\+/r0l0s0Pvip:EasyAlign*/[:;]\+/mrcrPvip:EasyAlign*/[:;]\+/mrcrr0l2108Goa+=1bb==#2ccc=3dddd=>4vip:EasyAlign=dll0s1r5id112Gopvip:EasyAlign/[:;]\+/msrrc**lm0rm3stl1dacidtd88Gvipjy116Gpvip:EasyAlign3\ ig['String']Pvip:EasyAlign3\ ig['string']
