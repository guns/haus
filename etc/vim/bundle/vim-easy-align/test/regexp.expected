" :source run.vim
" @a

apple;:banana  :: cake
data;;exchange :; format

apple ;: banana   ::   cake
 data ;; exchange :; format

apple ;:   banana :: cake
data  ;; exchange :; format

apple ;:____banana::cake
data  ;;____exchange:;format

apple ;  : banana   :  : cake
data  ;  ; exchange :  ; format

ap  pl  e;  :b  an  an  a:  :c  ak e
da  ta  ;;  ex  ch  an  ge  :;  fo  rm  at

ap ple;:banana::cake
da ta;;exchange:;format

apple???;:~~~banana???::~~~  cake
data???;;~~~ exchange???:;~~~format

apple;: banana::cake
data;;  exchange:;format

apple<<<;:>>>banana::cake
data <<<;;>>>exchange:;format

apple ;: banana::cake
data  ;; exchange:;format

apple;:banana  :: cake
data;;exchange :; format

apple ;: banana   :: cake
data  ;; exchange :; format

apple ;: banana   :: cake
data  ;; exchange :; format

apple ;: banana   :: cake
data  ;; exchange :; format

apple ;: banana   :: cake
data  ;; exchange :; format

apple ;: banana::cake
data  ;; exchange:;format

apple ; :banana::cake
data  ;  ; exchange: ; format

apple ; :banana::cake
data  ; ;exchange:;format

apple ; :banana::cake
data  ; ;exchange:;format

apple;:banana::cake
data;;exchange:;format

apple;:         banana: : cake
data;;exchange: ;format

apple;:banana: :cake
data;;exchange:;format

apple;:banana: :cake
data;;exchange:;format

apple;:         banana: : cake
data;;exchange: ;format

apple;:banana:: cake
data;;exchange: ;format

apple;:         banana: : cake
data;;exchange: ;format

apple;:         banana::cake
data;;exchange: ;format

```ruby
apple = 1                # comment not aligned
apricot = 'DAD' + 'F#AD'
banana = 'Gros Michel'   # comment 2
```

a()p()p()l()e();():()b()a()n()a()n()a():():()c()a()k()e(
d()a()t()a();();()e()x()c()h()a()n()g()e():();()f()o()r()m()a()t(

apple  ;: banana   ::  cake
 data  ;;exchange  :;format

apple ;:  banana  ::   cake
 data ;; exchange :; format

apple;:banana  ::cake
data ;;exchange:;format

apple   ;:   banana   ::     cake
data   ;;    exchange   :;   format

a+=         1
bb==#       2
ccc=        3
dddd=>      4

apple;:     banana::    cake
 data;;   exchange:;   format

```ruby
apple = 1       # comment not aligned
apricot = 'DAD' + 'F#AD'
banana = 'Gros  Michel'   # comment 2
```

```ruby
apple = 1              # comment not aligned
apricot = 'DAD'        + 'F#AD'
banana = 'Gros Michel' # comment 2
```

